src/v2/Apps/Order/__tests__/OrderApp.jest.tsx

```diff
@@ -571,6 +571,15 @@ describe("OrderApp", () => {
     expect(subject.find("ZendeskWrapper")).toHaveLength(1)
   })
 
+  it("does not show the Zendesk chat integration button if in a modal", () => {
+    const props = getProps()
+    const subject = getWrapper({
+      props: { ...props, location: { query: { isModal: true } } },
+    })
+
+    expect(subject.find("ZendeskWrapper")).toHaveLength(1)
```

Comment:
@lilyfromseattle - should this be `toHaveLength(0)` since we don't want it to be in the modal?